src/get-output.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import outputJson from "./output-json.js";
+import outputText from "./output-text.js";
+
+import type {ErrorCode} from "./error-codes.js";
+import type {Options, ILog, JsonItem} from "./types.js";
+
+type OutputFn = (
+    options: Options,
+    log: ILog,
+    jsonItems: Array<JsonItem>,
+    violationFileNames: Array<string>,
+) => ErrorCode;
```

Comment:
What do you think about moving this to `./types.js` and then using it to explicitly enforce the signature of the `outputJson` and `outputText` exports?

Reply:
Will do.